bin/stacks/routing-lambda-stack.ts

```diff
@@ -171,6 +244,8 @@ export class RoutingLambdaStack extends cdk.NestedStack {
       runtime: aws_lambda.Runtime.NODEJS_18_X,
       entry: path.join(__dirname, '../../lib/handlers/index.ts'),
       handler: 'quoteHandler',
+      vpc: vpc,
```

Comment:
is it possible to set vpc to only caching routing lambda first?

Reply:
updated to put both lambda under vpc as we discussed